test/controllers/responses_controller_test.rb

```diff
@@ -1,7 +1,73 @@
 require 'test_helper'
 
 class ResponsesControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    @user  = create(:user)
+    @admin = create(:admin)
+  end
+
+  test "response can only be destroyed by a logged in user" do
+    @survey   = create(:survey, user: @user)
+    @response = create(:response, survey: @survey)
+
+    delete survey_response_url(@survey, @response)
+    assert_redirected_to new_user_session_path
```

Comment:
Let's test that the number of records decreases here

Reply:
Uhm @spencerldixon in this case I don't think the number of records should decrease as it is meant to fail.